### DISCUSSION TITLE
Welcome to RPiList Discussions!

### DISCUSSION THREAD

**Discussion by RPiList** [RPiList/specials#1088] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TomfromBerlin** [comment]
Am I right in assuming that channels of the German public broadcaster have recently been blocked in the lists? Is that intentional? I whitelisted them and, lo and behold, I can access them again.

**Reply by yoshimo** [reply]
Update your lists, as written in #1116

**Reply by TomfromBerlin** [reply]
Thanx.
